pages/docs/reference/whatsnew14.md

```diff
@@ -1484,3 +1484,41 @@ artifact, which shades the bundled third-party libraries to avoid usage conflict
 If, for some reason, you need artifacts that depend on the unshaded `kotlin-compiler`, use the artifact versions with the 
 `-unshaded` suffix, such as `kotlin-scripting-jsr223-unshaded`. Note that this renaming affects only the scripting artifacts
 that are supposed to be used directly; names of other artifacts remain unchanged.
+
+
+
+## Kotlin/JS
+
+### New Gradle DSL
+The `kotlin.js` Gradle plugin comes with an adjusted Gradle DSL, which provides a number of new configuration options and is more closely aligned to the DSL used by the `kotlin-multiplatform` plugin. Some of the most impactful changes include:
+
+- Explicit toggles for the creation of executable files via `binaries.executable()`. Read more about the executing Kotlin/JS and its environment [here](js-project-setup.html#choosing-execution-environment).
+- Configuration of webpack's CSS and style loaders from within the Gradle configuration via `cssSupport`. Read more about using them [here](js-project-setup.html#configuring-css).
+- Improved management for npm dependencies, with mandatory version numbers or [semver](https://docs.npmjs.com/misc/semver#versions) version ranges, as well as support for _development_, _peer_, and _optional_ npm dependencies using `devNpm`, `optionalNpm` and `peerNpm`. Read more about dependency management for npm packages directly from Gradle [here](js-project-setup.html#npm-dependencies).
+- Stronger integrations for [Dukat](https://github.com/Kotlin/dukat), the generator for Kotlin external declarations. External declarations can now be generated at build time, or can be manually generated via a Gradle task. Read more about how to use the integration [here](js-modules.html#automatic-generation-of-external-declarations-with-dukat).
+
+### JS IR compiler backend
```

Comment:
Please write `back-end` with a hyphen (-) as everywhere else. In blog posts and what's new we just call them JS IR back-end – `compiler` is not necessary. Also I've added the word `new`. To make back-end sections consistent you can use the title **New JS IR back-end** if you don't mind. I have **New JVM IR back-end**.

Reply:
Adjusted!